vadim.martsun/task_4/src/main/java/Main.java

```diff
@@ -0,0 +1,42 @@
+import config.AppConfig;
+import lombok.extern.slf4j.Slf4j;
+import objects.Account;
+import objects.Transaction;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import random.RandomBankAccountService;
+import service.BankService;
+import service.transactionExplorer.TransactionExplorer;
+
+import java.util.List;
+
+@Slf4j
+public class Main {
+
+    private static void bankServiceDemo() throws Exception {
+        ApplicationContext context = new AnnotationConfigApplicationContext(AppConfig.class);
+        BankService bankService = (BankService) context.getBean("bankService");
```

Comment:
Better to user @Autowired for bankService instead of direct invocation of context